benchmark/micro_benchmark.cpp

```diff
@@ -147,7 +156,7 @@ void dense_iter_full_fwd_scan(benchmark::State &state) {
     for (auto i = 0; i < full_scan_multiplier; ++i) {
       std::uint64_t sum = 0;
       auto fn = [&sum](const unodb::visitor<typename Db::iterator> &v) {
-        sum += v.get_key();
+        sum += decode(v.get_key());
```

Comment:
@laurynas-biveinis it looks like decode() was added to keep the read of the internal key and update the sum.  

The key_decoder initialization is pretty light weight, but maybe we should change the sum to a sum of hashes of the encoded key?  I suspect the decode is cheaper, but we can't decode all keys once we get into variable length keys.

Reply:
Makes sense